cmd/ddev/cmd/commands.go

```diff
@@ -3,6 +3,13 @@ package cmd
 import (
 	"bufio"
 	"fmt"
+	"io/ioutil"
+	"os"
+	"path"
+	"path/filepath"
+	"runtime"
+	"strings"
+
```

Comment:
This change was not really intended but made by the linter of VS Code. And looking to other packages this looks like a best practise to place interal packages on the top and gh imports afterwards.